src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -26,16 +26,25 @@ object WdlExpression {
 
   val unaryOperators = Set("LogicalNot", "UnaryPlus", "UnaryNegation")
 
-  def evaluate(ast: AstNode, lookup: ScopedLookupFunction, functions: WdlFunctions): Try[WdlValue] = {
+  def interpolate(str: String, lookup: ScopedLookupFunction): String =
+    "\\$\\{([a-zA-Z0-9_]+)\\}".r.findAllIn(str).foldLeft(str) { (string, tag) =>
```

Comment:
regarding `\w` or `[[:word:]]`... I think maybe it'd be better just to copy/paste the "identifier" regex from WdlParser.java (`[a-zA-Z]([a-zA-Z0-9_])*`).  Ideally we'd just reference this regex but it's not exposed very well.

Really I wanted to create a parser entry point in the parser itself for parsing interpolation